src/items/__tests__/item.service.spec.ts

```diff
@@ -256,5 +256,37 @@ describe('ItemService', () => {
         expect(actual).toEqual(expected);
       });
     });
+    describe('and the execution is unsuccessful', () => {
+      it('throw a NotFoundException when item is not found', async () => {
```

Comment:
try to make the reading easier. When you put the it clause, you put 'should' first and then describe what should happen ;)